src/collector.ts

```diff
@@ -105,9 +105,33 @@ class NaiveGomodParser {
     dependencies: Array<IDependency>;
 
     static parseDependencies(contents:string, goImports: Set<string>): Array<IDependency> {
+        let replaceDependencies = new Map();
         let goModDeps = contents.split("\n").reduce((dependencies, line, index) => {
-            // Ignore "replace" lines
-            if (!line.includes("=>")) {
+            if (line.includes("=>")) {
```

Comment:
@dvandra Can this be simplified? I prefer to have a separate function to return the replace map.

Also add unit test.

Reply:
+1